### PR TITLE
display Fetch Job stats via icon click

### PR DESCRIPTION
**Pull request by peterVG**
Making use of the Fetch Job statistics that are already being stored in the database, this commit displays them on the Storage Service template. Rather than clutter the screen with stats for multiple Fetch Jobs, it only reveals the stats when clicking an Info icon. Clicking the icon again re-hides the stats info.

Connected to https://github.com/artefactual-labs/AIPscan/issues/47

### CODE REVIEW

File: AIPscan/Aggregator/templates/storage_service.html
Comment (tw4l) [inline AIPscan/Aggregator/templates/storage_service.html:54]: I'd recommend using Bootstrap tags and classes instead of a custom `style` tag to manage the styles. So for instance,  inside this `<div>` you could do something like
```
<span class="text-muted">
  <small>
    {{ mets_fetch_job.total_aips }} AIPs<br>
    {{ mets_fetch_job.total_sips }} SIPs<br>
    {{ mets_fetch_job.total_dips }} DIPs<br>
    {{ mets_fetch_job.total_replicas }} replicated AIPs<br>
    {{ mets_fetch_job.total_replicas }} deleted AIPs
  </small>
</span>
```
We do that elsewhere in the code base as well, e.g.: https://github.com/artefactual-labs/AIPscan/blob/main/AIPscan/Reporter/templates/report_largest_files.html#L74. The main advantage this change would give is if we want to change what our `text-muted` style looks like, we do so in one place and have it apply to the whole project (and full transparency, on my mind because I think the low contrast might be a bit of an accessibility issue, and so we might want to find a less-muted shade of grey).

File: AIPscan/Aggregator/templates/storage_service.html
Comment (tw4l) [inline AIPscan/Aggregator/templates/storage_service.html:53]: For HTML5, the preferred line break tag is `<br>`, no trailing space/slash needed: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br

File: AIPscan/Aggregator/templates/storage_service.html
Comment (tw4l) [inline AIPscan/Aggregator/templates/storage_service.html:58]: Nit: We often call these "replicated AIPs", but it's a bit ambiguous whether that means the original or the replica. I think it might be more accurate or at least unambiguous to call them "AIP replicas".

File: AIPscan/Aggregator/templates/storage_service.html
Comment (tw4l) [inline AIPscan/Aggregator/templates/storage_service.html:85]: Succinct, readable, and effective - nice job!

One minor nit/linting issue is that it'd be great to have opening and closing curly braces aligned - these are fairly small closures so not an immediate practical need maybe, but having them properly aligned really helps with readability when code grows. In practical terms I think it would just mean moving the closing `});`s here two spaces to the left.

Comment (tw4l) [reply AIPscan/Aggregator/templates/storage_service.html:85]: This is the sort of thing that we will ideally eventually have a JS linter to help spot (and potentially even auto-correct for us, like `black` does for our Python). And it's absolutely a nit, so I'll leave it up to you to decide if it's worth the time.

**Review by tw4l — APPROVED** [review]
Hi @peterVG - this is looking great! I've left some suggestions and a few nits, but this is already solid so I'm approving and you can make those changes before squashing and merging. Nice work! :muscle:

File: AIPscan/Aggregator/templates/storage_service.html
Comment (peterVG) [reply AIPscan/Aggregator/templates/storage_service.html:54]: Yes, custom inline CSS is a bad temp hack. Thanks for this cleanup suggestion. Done.

File: AIPscan/Aggregator/templates/storage_service.html
Comment (peterVG) [reply AIPscan/Aggregator/templates/storage_service.html:53]: themoreyouknow

Comment (peterVG) [reply AIPscan/Aggregator/templates/storage_service.html:53]: I learned basic HTML back in the 90's and this was considered very important (yet totally irrelevant in the end) practice.

File: AIPscan/Aggregator/templates/storage_service.html
Comment (peterVG) [reply AIPscan/Aggregator/templates/storage_service.html:58]: +1 Yes agreed.

Comment (peterVG) [reply AIPscan/Aggregator/templates/storage_service.html]: Also, great nit-picking. I do this kind of semantic hair splitting all the time and appreciate it when others do it too. Accuracy is in our field is important. Starting with how we describe the tools that run our systems.

File: AIPscan/Aggregator/templates/storage_service.html
Comment (peterVG) [reply AIPscan/Aggregator/templates/storage_service.html:89]: I'll take any suggestions on improving the readability of Javascript / JQuery, lol. I've moved the bracket as suggested. Have we shortlisted JS Linters yet? I'll gladly install one.

**Pull request merged**
